client/extensions/woocommerce/app/settings/email/email-settings/components/internal-notification.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import CompactFormToggle from 'components/forms/form-toggle/compact';
+import FormTextInput from 'components/forms/form-text-input';
+import FormLabel from 'components/forms/form-label';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+
+const InternalNotification = ( { item, recipient, checked, onToggle, onChange } ) => (
+	<Card className="components__notification-component">
```

Comment:
Relate to the comment about the `List` components in `email-settings/index.js`, this probably needs to be a `ListItem` or something.